[2023-09-11 11:59:33,870] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: listing_4_15.extract_gz scheduled__2023-09-11T02:00:00+00:00 [queued]>
[2023-09-11 11:59:33,885] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: listing_4_15.extract_gz scheduled__2023-09-11T02:00:00+00:00 [queued]>
[2023-09-11 11:59:33,885] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-09-11 11:59:33,885] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-09-11 11:59:33,885] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-09-11 11:59:33,900] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): extract_gz> on 2023-09-11 02:00:00+00:00
[2023-09-11 11:59:33,907] {standard_task_runner.py:52} INFO - Started process 1179 to run task
[2023-09-11 11:59:33,910] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'listing_4_15', 'extract_gz', 'scheduled__2023-09-11T02:00:00+00:00', '--job-id', '478', '--raw', '--subdir', 'DAGS_FOLDER/listing_4_15.py', '--cfg-path', '/tmp/tmp6vajci1_', '--error-file', '/tmp/tmp3d0x85gg']
[2023-09-11 11:59:33,911] {standard_task_runner.py:80} INFO - Job 478: Subtask extract_gz
[2023-09-11 11:59:34,046] {task_command.py:371} INFO - Running <TaskInstance: listing_4_15.extract_gz scheduled__2023-09-11T02:00:00+00:00 [running]> on host b18943745c0d
[2023-09-11 11:59:34,141] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=listing_4_15
AIRFLOW_CTX_TASK_ID=extract_gz
AIRFLOW_CTX_EXECUTION_DATE=2023-09-11T02:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-09-11T02:00:00+00:00
[2023-09-11 11:59:34,142] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-09-11 11:59:34,143] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'gunzip --force /tmp/wikipageviews.gz']
[2023-09-11 11:59:34,158] {subprocess.py:85} INFO - Output:
[2023-09-11 11:59:34,586] {subprocess.py:92} INFO - 
[2023-09-11 11:59:34,586] {subprocess.py:92} INFO - gzip: /tmp/wikipageviews.gz: invalid compressed data--format violated
[2023-09-11 11:59:34,596] {subprocess.py:96} INFO - Command exited with return code 1
[2023-09-11 11:59:34,618] {taskinstance.py:1909} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 195, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-09-11 11:59:34,622] {taskinstance.py:1420} INFO - Marking task as FAILED. dag_id=listing_4_15, task_id=extract_gz, execution_date=20230911T020000, start_date=20230911T115933, end_date=20230911T115934
[2023-09-11 11:59:34,641] {standard_task_runner.py:97} ERROR - Failed to execute job 478 for task extract_gz (Bash command failed. The command returned a non-zero exit code 1.; 1179)
[2023-09-11 11:59:34,686] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-09-11 11:59:34,737] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
